spec/spec_helper.rb

```diff
@@ -2,3 +2,10 @@
   require 'simplecov'
   SimpleCov.start
 end
+
+VCR.configure do |config|
+  config.ignore_hosts 'codeclimate.com' if ENV['CI']
+  config.cassette_library_dir = File.join(ManageIQ::Providers::Lenovo::Engine.root, 'spec/vcr_cassettes')
+end
```

Comment:
I'm wondering if VCR configuration should be in the generator, because not all providers will use VCR